.eslintrc.js

```diff
@@ -1,5 +1,34 @@
 const { merge } = require('lodash')
 
 module.exports = merge(require('@tstt/eslint-config/index.js'), {
-  globals: { Meteor: 'readonly', i18n: 'readonly' },
+ env: {
```

Comment:
I think some are already present here

https://github.com/techstertech/eslint-config/blob/main/.eslintrc.js

Reply:
I removed the duplicate ones